addons/delivery/models/delivery_carrier.py

```diff
@@ -9,6 +9,18 @@
 from .delivery_request_objects import DeliveryCommodity, DeliveryPackage
 
 
+class DeliveryZipPrefix(models.Model):
+    """ Zip prefix that a delivery.carrier will deliver to. """
+    _name = 'delivery.zip.prefix'
+    _description = 'Delivery Zip Prefix'
+
+    name = fields.Char('Prefix', required=True, translate=True)
+
+    _sql_constraints = [
+        ('name_uniq', 'unique (name)', "Prefix already exists!"),
+    ]
+
+
```

Comment:
Do you mind placing this class in a separate file?
The file is big enough, and IMHO, it's way clearer if we keep the files split by models/classes :).

Reply:
Done.